dkg-test-suite/tests/wrappingFeeUpdateProposal.test.ts

```diff
@@ -0,0 +1,126 @@
+import { jest } from '@jest/globals';
+import 'jest-extended';
+import {
+	WrappingFeeUpdateProposal,
+	ChainIdType,
+	encodeFunctionSignature,
+	encodeWrappingFeeUpdateProposal,
+	ethAddressFromUncompressedPublicKey,
+	fetchDkgPublicKey,
+	registerResourceId,
+	sleep,
+	startStandaloneNode,
+	waitForEvent,
+	waitUntilDKGPublicKeyStoredOnChain,
+	AnchorUpdateProposal,
+	encodeUpdateAnchorProposal,
+} from '../src/utils';
+import { LocalChain } from '../src/localEvm';
+import { ChildProcess } from 'child_process';
+import { ethers } from 'ethers';
+import { Anchors, Bridges } from '@webb-tools/protocol-solidity';
+import { MintableToken, GovernedTokenWrapper, TokenWrapperHandler } from '@webb-tools/tokens';
+import { ApiPromise, Keyring } from '@polkadot/api';
+import { provider } from '../src/utils';
+import { ACC1_PK, ACC2_PK, BLOCK_TIME, SECONDS } from '../src/constants';
+import {hexToNumber, numberToHex, u8aToHex} from '@polkadot/util';
+import { Option } from '@polkadot/types';
+import { HexString } from '@polkadot/util/types';
+import { signAndSendUtil } from '../src/evm/util/utils';
+import {
+	aliceNode,
+	bobNode,
+	localChain,
+	polkadotApi,
+	signatureBridge,
+	wallet1,
+	wallet2,
+	charlieNode,
+	localChain2, executeAfter
+} from './utils/util';
+
+describe('Wrapping Fee Update Proposal', () => {
+
+	test('should be able to sign wrapping fee update proposal', async () => {
```

Comment:
Do another negative test. Where the wrapping fee is over 100. and it should fail ofc.